tests/test_examples.py

```diff
@@ -33,3 +35,10 @@ def test_examples(example_name):
     else:
         with pytest.raises((ValidationError, ValueError)):
             PackageManifest.parse_obj(example_json).dict()
+
+
+def test_open_zeppelin_contracts():
```

Comment:
the other tests pull from Github directly.

but the manifest I compile is slightly different in that I removed `package.json` before compiling it.

maybe we need an `ignore_files:` keyword in the `dependencies:` item?

Reply:
Potentially, or `DependencyAPI` should be aware of these "Extra" files and remove them before making a package manifest

Reply:
Oh yeah we already do that actually.
This was just happening because I was compiling the OZ contracts directly as a project.